content/pages/talks.html

```diff
@@ -1,16 +1,44 @@
 ---
-$title@: Talks
+$title@: Presentations
 $hidden: true
+$path: /presentations/
 ---
 
 <div class="header about-header">
-    <h2>{{doc.title}}</h2>
+    <h2>Presentations</h2>
 </div>
+{% set highlighted = 'talks' %}
 {% include 'partials/about_nav.html' %}
-    
+
+<div class="doc-nav sub">
+    <ul>
+        <li><a href="#talks"><h5>Talks</h5></a></li>
+        <li><a href="#Posters"><h5>Posters</h5></a></li>
```

Comment:
can we keep a style of non caps for these id's?

Reply:
changed